pkg/controllers/user/alert/deployer/notification_template.go

```diff
@@ -105,8 +105,12 @@ Project Name: {{ .Labels.project_name}}{{ end }}
 {{- if .Labels.pod_name }}
 Pod Name: {{ .Labels.pod_name}}{{ else if .Labels.pod -}}Pod Name: {{ .Labels.pod}}{{ end }}
 Expression: {{ .Labels.expression}}
+{{- if .Labels.threshold_value }}
+Description: The configured event happened for ({{ .Labels.duration}}): expression matched, datapoint value is {{ .Annotations.current_value}}
+{{- else}}
```

Comment:
Looks like we print `Threshold Crossed ...` when `threshold_value` is empty. Should we do it the other way?